frontend/src/app/users/users-profile/users-profile.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
```

Comment:
The user overview and the global user overviews should have links to the user profiles when clicking on a new button on an individual user.

Reply:
what do you mean by that? you can now get to the view via the project user overview and the user overview in the settings - are there any other?

Reply:
No, these are the only two :)